src/components/sections/section.tsx

```diff
@@ -78,6 +81,9 @@ const useInputGroupStyle = StyleSheet.createThemeHook((theme: Theme) => ({
     marginHorizontal: theme.spacings.medium,
     marginBottom: theme.spacings.small,
   },
+  container__fullPadded: {
+    margin: theme.spacings.medium,
```

Comment:
Will this be correct? topPadded + bottomPadded are `theme.spacings.large`. Also instead of creating its own padding-style, could maybe re-use the other two?

Reply:
I couldn't find a perfect solution here. Using padded + topPadded made the top padding too large as it was `theme.spacings.large`. I can make the change @mikaelbr suggested below. But maybe in it's own PR as it will probably touch lots of files?